pyproject.toml

```diff
@@ -161,9 +161,9 @@ ignore = [
     "S105",   # flake8-bandit: hardcoded password
     "S106",   # flake8-bandit: hardcoded password
     "S603",   # flake8-bandit: subprocess-without-shell-equals-true
-    "S320",   # flake8-bandit: xml parsing vulnerable to XML attacks
     "UP007",  # forbid use of typing.Union and typing.Optional
     "UP015",  # Unnecessary mode="r" argument in open()
+    "UP045",  # Forced use of "|" instead of typing.Optional and typing.Union
```

Comment:
While I prefer the new syntax, I think it's overkill if our CI forbids the old one.

Reply:
But it's easy enough to do, and if you prefer it, why not just enforce it? :)  
(I prefer it too, as you may have guessed)